ShopLens/VoicedText/TextVoicers/TextVoicer.cs

```diff
@@ -12,16 +12,34 @@ public class TextVoicer
 
         //The speed at which the voicer says things.
         //By default the speed is normal.
-        public int SpeedOfVoicer { get; set; }
+        private int speedOfVoicer;
+        public int SpeedOfVoicer
+        {
+            get { return speedOfVoicer; }
+            set { speedOfVoicer = value; }
+        }
+
+        //Should we use auto properties instead (the IDE suggests to do so, 
+        //but that first capital letter naming convention, man)?
```

Comment:
Yes, we should - naming in camelCase is for LOCAL variables only. All properties with getters and setters should be named in PascalCase.